spawner:
  rate: 25                   # Really slow for grasping one object at a time
  models:
    - mug
    # - wood_block
    # - master_chef_can
    # - bleach_cleanser
    # - mustard_bottle
  # randOrientation: true
  randOrientation: false        # For now to test the grasping
